nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -86,7 +94,14 @@ def input_example(self, max_batch=1, max_dim=256):
         dev = next(self.parameters()).device
         input_example = torch.randn(max_batch, self._feat_in, max_dim).to(dev)
         input_example_length = torch.randint(1, max_dim, (max_batch,)).to(dev)
-        return tuple([input_example, input_example_length])
+
+        if self.export_cache_support:
```

Comment:
check hasattr(self, 'export_cache_support') and self.export_cache_support.

Reply:
done.